bot/exts/evergreen/emojis.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+
+from discord import Color, Embed, Emoji
+from discord.ext import commands
+
+from bot.utils.time import time_since
+
+
+class Emojis(commands.Cog):
+    """Has commands related to emojis."""
+
+    @commands.group(name="emoji", invoke_without_command=True)
+    async def emojis_group(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """A group of commands related to emojis."""
+        # No parameters = same as invoking info subcommand
+        await ctx.invoke(self.info_command, emoji)
```

Comment:
Not required to change it, but what about using `await self.info_command(...)` instead invoking?

Reply:
We had a discussion about this in #dev-core before, and came to the conclusion that we should use the `invoke` method instead. It does some extra things that awaiting the command itself does not do.